homeassistant/helpers/config_validation.py

```diff
@@ -711,7 +711,7 @@ def deprecated(
         - No warning if neither key nor replacement_key are provided
             - Adds replacement_key with default value in this case
     """
-    module = inspect.getmodule(inspect.stack()[1][0])
+    module = inspect.getmodule(inspect.stack(context=1)[1].frame)
```

Comment:
What do we use the context line for here?

Reply:
Oops, it should be:
```suggestion
    module = inspect.getmodule(inspect.stack(context=0)[1].frame)
```